docs/sphinx/reference-libobs-util-platform.rst

```diff
@@ -478,3 +478,11 @@ Other Functions
 .. function:: uint64_t os_get_proc_virtual_size(void)
 
    Returns the virtual memory size of the current process.
+
+---------------------
+
+.. function:: bool os_get_emulation_status(void)
+
+   Returns true if the current process is a x64 binary and is being emulated or translated
+   by the host operating system. On macOS, it returns true when a x64 binary is 
+   being translated by Rosetta and running on Apple Silicon Macs.
```

Comment:
This might need a note that it currently isn't implemented on Windows and Linux and always returns false on those systems.

Reply:
good idea

Reply:
Wait it is implementled on Windows, just not here.

Reply:
I’d say we should still have this note as long as the Windows implementation isn’t in (which is out of scope for this particular PR). The PR that implements it on Windows would remove the Windows part again.

Reply:
Done.